src/services/telemetry.ts

```diff
@@ -11,33 +11,33 @@ type IasqlOperationType = typeof IasqlOperationType[keyof typeof IasqlOperationT
 
 const singleton = config.telemetry ? Amplitude.init(config.telemetry.amplitudeKey) : undefined;
 
-export async function logDbConnect(dbId: string, dbAlias: string, uid: string, email: string) {
+export async function logDbConnect(dbId: string, userProps: any) {
```

Comment:
nit: maybe add `todo` to type this

Reply:
hmm it varies per event, but maybe I can type it now